src/hackerspace_online/settings/base.py

```diff
@@ -546,3 +546,7 @@ def POSTMAN_NOTIFICATION_APPROVAL(u): return u.profile.get_messages_by_email
 TENANT_MODEL = "tenant.Tenant"
 
 DEFAULT_FILE_STORAGE = 'tenant_schemas.storage.TenantFileSystemStorage'
+
+
+DEFAULT_TENANT_SUPERUSER_USERNAME = 'admin'
+DEFAULT_TENANT_SUPERUSER_PASSWORD = 'admin1234'
```

Comment:
Why is this in the base settings file?  This should be in the local settings file as it is specific to local development and will definitely not be common between local vs production.

Reply:
Also, settings should begin with their qpp name, so this should be TENANT_DEFAULT_SUPERUSER_USERNAME for example.